images/win/scripts/Installers/Install-Git.ps1

```diff
@@ -8,6 +8,7 @@ Import-Module -Name ImageHelpers
 # Install the latest version of Git which is bundled with Git LFS.
 # See https://chocolatey.org/packages/git
 choco install git -y --package-parameters="/GitAndUnixToolsOnPath /WindowsTerminal /NoShellIntegration"
+choco install hub
```

Comment:
Please update documentation to provide hub version

Reply:
Done: https://github.com/andy-mishechkin/virtual-environments/commit/9eb8d744a7f31c7d839aa8ea6a5fc21ca0147e5c
Builds:
https://github.visualstudio.com/virtual-environments/_build/results?buildId=62031&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=486c0909-f67d-5748-80b3-bb1d531e7b43
https://github.visualstudio.com/virtual-environments/_build/results?buildId=62030&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=486c0909-f67d-5748-80b3-bb1d531e7b43
https://github.visualstudio.com/virtual-environments/_build/results?buildId=62028&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=486c0909-f67d-5748-80b3-bb1d531e7b43
https://github.visualstudio.com/virtual-environments/_build/results?buildId=62029&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=486c0909-f67d-5748-80b3-bb1d531e7b43